### DISCUSSION TITLE
Partial trigger deployment

### DISCUSSION THREAD

**Discussion by proverbs53** [Azure-Player/azure.datafactory.tools#436] [Ideas]
Looking at the code, it appears still an approach is used stopping all triggers. Did you consider moving to an approach where triggers are stopped only if they are altered and are in a started state, cf. https://techcommunity.microsoft.com/blog/azuredatafactoryblog/cicd-improvements-related-to-pipeline-triggers-deployment/3605064?

**Comment by NowinskiK** [comment]
This approach is already implemented, check the documentation and this deployment option:
`TriggerStopMethod` - determines which triggers should be stopped.
Available values: `AllEnabled` (default) | `DeployableOnly`

**Comment by proverbs53** [comment]
Got it (https://github.com/Azure-Player/azure.datafactory.tools/blob/master/private/Stop-Triggers.ps1#L29) , thanks a lot!
